src/main/java/com/synopsys/integration/alert/channel/email/descriptor/EmailDistributionUIConfig.java

```diff
@@ -37,18 +37,15 @@
 
 @Component
 public class EmailDistributionUIConfig extends UIConfig {
-    public static final String KEY_SUBJECT_LINE = "subject.line";
-    public static final String KEY_PROJECT_OWNER_ONLY = "project.owner.only";
-    public static final String KEY_EMAIL_ADDRESSES = "email.addresses";
 
     @Override
     public UIComponent generateUIComponent() {
         return new UIComponent("Email", "email", EmailGroupChannel.COMPONENT_NAME, "envelope", setupFields());
     }
 
     public List<ConfigField> setupFields() {
-        final ConfigField subjectLine = new TextInputConfigField(KEY_SUBJECT_LINE, "Subject Line", false, false);
-        final ConfigField projectOwnerOnly = new CheckboxConfigField(KEY_PROJECT_OWNER_ONLY, "Project Owner Only", false, false);
+        final ConfigField subjectLine = new TextInputConfigField(EmailDescriptor.KEY_SUBJECT_LINE, "Subject Line", false, false);
+        final ConfigField projectOwnerOnly = new CheckboxConfigField(EmailDescriptor.KEY_PROJECT_OWNER_ONLY, "Project Owner Only", false, false);
```

Comment:
The constructors for ConfigFields seem to be loaded with more than they need. We should be able to remove some parameters for subclasses and replace them with defaults in the super call.

For example, we shouldn't need to specify that a CheckboxConfigField is not sensitive every time we create it. I don't think a checkbox will ever be sensitive. I know you didn't implement that, but while you're in there, it might be a worthwhile change.